yt_dlp/extractor/lumni.py

```diff
@@ -0,0 +1,23 @@
+from .common import InfoExtractor
+from .francetv import FranceTVIE
+
+
+class LumniIE(InfoExtractor):
```

Comment:
I think it would be better to move this extractor to francetv.py, they're closely related

Reply:
Thanks for the help! I had made this as a separate extractor because I interpreted the comment at https://github.com/yt-dlp/yt-dlp/issues/6202#issuecomment-1425827384 to mean that a completely separate extractor was needed. So you mean I could simply move this code into something like a LumniTVSiteIE class in extractors/francetv.py?

Reply:
separate file's fine